app/services/allocation_importer_service.rb

```diff
@@ -0,0 +1,46 @@
+require "csv"
+
+class AllocationImporterService
+  attr_reader :path_to_csv
+
+  def initialize(path_to_csv:)
+    @path_to_csv = path_to_csv
+  end
+
+  def execute
```

Comment:
Aren't we migrating to using `call`? @gpeng was going to start enforcing this I thought. 😛

Reply:
I prefer `call`, README says to use `execute` so maybe we should change the README

Reply:
We are/were but haven't as yet so I let it slip. I've seen this @asmega flavour elsewhere so it's currently as consistent as anything else 😆